pkg/cpuid/cpuid_amd64.s

```diff
@@ -16,9 +16,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-TEXT ·cpuidAsm(SB), 4, $0   // 4 = NOSPLIT
-    MOVL    eax_arg+0(FP), AX
-    MOVL    ecx_arg+4(FP), CX
+TEXT ·cpuidAsm(SB), 4, $0-24   // 4 = NOSPLIT
```

Comment:
Why do we add a `-24` here?

Reply:
That was to fix this problem:
```
$ go vet ./pkg/cpuid/
# sigs.k8s.io/node-feature-discovery/pkg/cpuid
pkg/cpuid/cpuid_amd64.s:19:1: [amd64] cpuidAsm: wrong argument size 0; expected $...-24
```

But, good question! There was a more proper way to fix this with `#include "textflag.h"`. Changed accordingly